lib/providers/dashboards_provider.dart

```diff
@@ -5,45 +5,35 @@ class DashboardsProvider with ChangeNotifier {
 
   final List<DashboardTab> _dashboardTabs = [
     DashboardTab(
-      title: 'dashboard',
+      title: 'Dashboard',
```

Comment:
Can we move user-visible labels to the language file.

Reply:
will be added to gitignore in change-request-part1 PR

Reply:
nice thanks